src/NuGet.Core/NuGet.Protocol/Resources/RawSearchResourceV3.cs

```diff
@@ -4,15 +4,13 @@
 using System;
 using System.Collections.Generic;
 using System.Globalization;
-using System.IO;
 using System.Linq;
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using NuGet.Protocol.Core.Types;
-using NuGet.Protocol.Model;
 
 namespace NuGet.Protocol
 {
```

Comment:
Did we mark this class as Obsolete?

Reply:
No, because if do that then I have to make change to RawSearchResourceV3Provider or delete it. 
Is it ok to delete it RawSearchResourceV3Provider?
![image](https://user-images.githubusercontent.com/8766776/83470172-6e10b780-a436-11ea-82e7-cffe6efa7363.png)